tsl/src/reorder.c

```diff
@@ -794,6 +794,7 @@ copy_heap_data(Oid OIDNewHeap, Oid OIDOldHeap, Oid OIDOldIndex, bool verbose,
 			case HEAPTUPLE_RECENTLY_DEAD:
 				tups_recently_dead += 1;
 				/* fall through */
+				__attribute__((fallthrough));
```

Comment:
Better to use the `TS_FALLTHROUGH` macro instead of this.

Reply:
@nikkhils thanks that looks good, I force pushed an update to the PR